lib/zendesk_apps_tools/command_helpers.rb

```diff
@@ -14,6 +14,10 @@ module CommandHelpers
     include ZendeskAppsTools::Directory
     include ZendeskAppsTools::PackageHelper
 
+    def self.included(base)
```

Comment:
sorry, I misunderstood here. `Command` includes `CommandHelpers`, not directly `Common` - this needs to be in both places so that classes that just include `Common` also get the ClassMethods.